src/bramble-live-dev-cache-sw.js

```diff
@@ -5,6 +5,24 @@
  * which is named vfs/project/root.
  */
 
+// String comes from src/filesystem/impls/filer/UrlCache.js
+var liveDevUrlRegex = /thimble-sw-vfs-cached-url\//;
+
+function custom500() {
+    "use strict";
+
+    var body = "<!doctype html><title></title><p>There was an error serving your content. Try restarting your web browser to clear your cache.";
+    var init = {
+        status: 500,
+        statusText: "Thimble live dev server failed to find cached URL",
+        headers: {
+            "Content-Type": "text/html"
```

Comment:
Do we also need to tell it not to cache this response? i.e. set a cache control header here?